index.html

```diff
@@ -6,15 +6,47 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="description"
-  content="full description of the app">
+        content="Software Engineer, Graphic Designer, Producer, Ramoun helps you remove obstcles to achieve your goals. Specializing in Full Stack Web Development, Logo Design & Music Producing">
   <meta name="robot" content="index,follow">
-  <link rel="stylesheet" href="styles/reset.css">
-  <link rel="stylesheet" href="styles/debug.css">
-  <title>Document | Description</title>
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/layout.css">
+  <link rel="stylesheet" href="css/-debug.css">
+  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <title>Ramoun | Software Engineer</title>
 </head>
 
 <body>
-  <h1>Hello, World!</h1>
+  <!-- header/begin -->
+  <header class="main-header bg">
+    <div class="wrapper">
+      <div id="logo">
+        <img src="./images/darwin.jpg" alt="">
+        <a href="./">Ramoun</a>
+      </div>
+      <nav>
+        <img src="./images/menu.svg" alt="">
+      </nav>
+    </div>
+  </header>
+  <!-- header/end -->
+
+  <!-- intro/begin -->
+  <section class="intro hero bg">
+    <div class="wrapper">
+      <h1>I’m&nbsp;Ramoun Glad&nbsp;to&nbsp;see you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a href="#">let's connect</a>
+      <div class="social">
+        <a href="https://twitter.com/ramoun16"><img src="./images/social/twitter.svg" alt=""></a>
+        <a href="https://www.linkedin.com/in/ramoun"><img src="./images/social/linkedin.svg" alt=""></a>
+        <a href="https://medium.com/@omartarekramoun"><img src="./images/social/medium.svg" alt=""></a>
+        <a href="https://github.com/mrramoun"><img src="./images/social/github.svg" alt=""></a>
+        <a href="https://angel.co/u/omar-ramoun"><img src="./images/social/vector.svg" alt=""></a>
+      </div>
```

Comment:
- [ ] This is great to add social media icons to your project, However, it could be nicer to wrap all the icons in an unordered list `<ul>` because this is supposed to be a list of icons, see the example below: 👇 
```
  <ul>
    <li>
        <a href="https://twitter.com/ramoun16"><img src="./images/social/twitter.svg" alt=""></a>
    </li>
    <li>
        <a href="https://github.com/mrramoun"><img src="./images/social/github.svg" alt=""></a>
    </li>
  </ul>
```
I promise this is going to make your code more clean and sematic 👌  😉